frontend/src/api/index.ts

```diff
@@ -45,3 +64,37 @@ export const fetchCourse = async (
     return null;
   }
 };
+
+export const searchCourses = async (
+  searchQuery: string,
+  minIndex: number = 0,
+  maxIndex: number = 10
+): Promise<ScheduleCourse[]> => {
+  const courses: ScheduleCourse[] = [];
+  const proxyurl = "https://cors-anywhere.herokuapp.com/";
+  const response = await fetch(
+    proxyurl +
+      `https://searchneu.com/search?query=${searchQuery}&termId=202130&minIndex=${minIndex}&maxIndex=${maxIndex}`,
```

Comment:
Should we have a helper to get the current term id (based on current day)?

Reply:
yea definitely - I'm hoping the "idea of a course" service where termIds don't matter will be ready before we need that though :)